packages/wix-storybook-utils/src/loader/prepareStory/prepareStory.js

```diff
@@ -0,0 +1,119 @@
+const types = require('@babel/types');
+const pathLib = require('path');
+const visit = require('react-autodocs-utils/src/parser/visit');
+const parse = require('react-autodocs-utils/src/parser/parse');
+const print = require('react-autodocs-utils/src/parser/print');
+const get = require('react-autodocs-utils/src/get');
+
+const prepareStory = (storyConfig, sourcePath) => source =>
+  new Promise((resolve, reject) => {
+    const isError = !source && !storyConfig && !sourcePath
+    return isError
+    ? reject('ERROR: unable to prepare story, both `storyConfig` and `source` must be provided')
+    : resolve(source)
+  })
+
+    .then(parse)
+
+    .then(ast => {
+      let isES5 = true;
+
+      visit(ast)({
+        ExportDefaultDeclaration() {
+          isES5 = false;
+          return false;
+        },
+      });
+
+      if (isES5) {
+        // add requires
+        ast.program.body.unshift(parse('const { storiesOf } = require("@storybook/react")'));
+        ast.program.body.unshift(parse('const story = require("wix-storybook-utils/Story").default'));
+      } else {
+        // add imports
+        ast.program.body.unshift(parse('import { storiesOf } from "@storybook/react"'));
+        ast.program.body.unshift(parse('import story from "wix-storybook-utils/Story"'));
+      }
+
+      return ast;
+    })
+    .then(ast => {
+      // TODO: this is not too good, unfortunatelly, i cant return
+      // rejected promise from within visitor, babylon complains
+      let error = null;
+
+      const configAST = parse(`(${JSON.stringify(storyConfig)})`);
+      let configProperties;
+
+      visit(configAST)({
+        ObjectExpression(path) {
+          const storiesOfProperty = types.objectProperty(types.identifier('storiesOf'), types.identifier('storiesOf'));
+
+          if(storyConfig.playgroundComponentsPath) {
+            const playgroundComponentsPath = pathLib.relative(sourcePath, storyConfig.playgroundComponentsPath);
+            const playgroundComponentsProperty = types.objectProperty(types.identifier('playgroundComponents'), types.identifier(`require('${playgroundComponentsPath}').default`));
```

Comment:
```suggestion
            const playgroundComponentsProperty = types.objectProperty(types.identifier('playgroundComponents'), parse(`require('${playgroundComponentsPath}').default`));
```

the value of a property is not an identifier, it's a function call. an easy way to get ast of that is to use `parse`